### ISSUE TITLE
error on getComponentID

### ISSUE THREAD

**Issue by juancabrera** [matthewwithanm/react-inlinesvg#6]
When I try to do this:
```html
<a className="logo" href="/"><Isvg src="/static/svg/logo-ginlane.svg">test</Isvg></a>
```

Got this error:
![screen shot 2015-03-02 at 5 39 41 pm](https://cloud.githubusercontent.com/assets/355253/6452352/25c7a374-c103-11e4-8982-c924ed50ecae.png)

And this is how it looks on the actual DOM:
```html
<a class="logo" href="/" data-reactid=".7ldt8ufdhc.0.0.0.0"><span class="isvg loading" data-reactid=".7ldt8ufdhc.0.0.0.0.0"></span></a>
```

**Comment by matthewwithanm** [comment]
Hm, what version of React?

**Comment by juancabrera** [comment]
@matthewwithanm 0.13.0-rc1 (actually need this version)

**Comment by matthewwithanm** [comment]
Ah, they probably removed that `_rootNodeID`. We probably don't really need an id unique to the component instance, though…some hash of the src or svg text should do just as well.

I'll try to get to this later this week, if nobody else submits a PR first.

**Comment by juancabrera** [comment]
ok cool — will take a look tomorrow morning, hopefully I can submit a PR.

**Comment by juancabrera** [comment]
please see #7

**Comment by matthewwithanm** [comment]
Closed by #7!
